docs/get_started.rst

```diff
@@ -130,7 +130,7 @@ Delete Stack
 
 Delete the stack::
 
-  $ sceptre delete-stack dev vpc
+  $ sceptre delete-stack dev vpc [--yes]
```

Comment:
@theseanything  the --help option for delete stack now shows that you should pass --yes for delete-stack. 

However, I'm not sure where in the documentation this should be placed - I don't see any clear location for it. (I.e. there aren't any sceptre-command  specific pages for this anywhere at the moment in the docs.)